lib/pdftron_flutter.dart

```diff
@@ -62,6 +62,12 @@ class PdftronFlutter {
         Functions.importAnnotations, <String, dynamic>{Parameters.xfdf: xfdf});
   }
 
+  /// Merges the given XFDF annotation string to the current document.
```

Comment:
Can you also please expose to `document_view.dart`
Follow the way importAnnotations is declared there

Reply:
found out changes were not changing due to mismatch between local/remote branch 
- updated changes (the old changes were also not being pushed either too so that has been updated)